config/initializers/stripe.rb

```diff
@@ -1,6 +1,6 @@
 Rails.configuration.stripe = {
-  publishable_key: ENV['STRIPE_PUBLISHABLE_KEY'],
-  secret_key: ENV['STRIPE_SECRET_KEY']
+  publishable_key: 'pk_test_a16qIgYkFnx8jVyXFffMLHhL',
```

Comment:
This shouldn't be committed to version control. It should only exist in application.yml. If someone discovered these keys (and people are clever enough to write scrappers to search for them) it could compromise your Stripe account. Let's chat about this over Slack if you're not sure what I'm talking about. :)